pkg/flowaggregator/flowaggregator.go

```diff
@@ -416,7 +428,7 @@ func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 			expireTimer.Stop()
 			return
 		case <-expireTimer.C:
-			if fa.exportingProcess == nil {
+			if fa.externalFlowCollectorAddr != "" && fa.exportingProcess == nil {
```

Comment:
I think that answers my earlier question about whether the flow aggregator can work properly when `externalFlowCollectorAddr` is empty.